controllers/search_controller.rb

```
@@ -8,61 +8,53 @@ class SearchController < ApplicationController
       process_search
     end
 
+    get '/content' do
+      process_search
+    end
+
+    get '/metadata' do
+      query = get_query(params)
+      options = get_ontology_metadata_search_options(params)
+      page, page_size = page_params
```

RuboCop (1.73)

[warning] Useless assignment to variable - `page`. Use `_` or `_page` as a variable name to indicate that it won't be used.

(Lint/UselessAssignment)